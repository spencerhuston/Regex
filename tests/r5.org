No match
Match
Match
No match
No match
No match
