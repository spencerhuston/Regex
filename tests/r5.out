Match
Match
Match
No match
Match
Match
